src/components/Editor.tsx

```diff
@@ -18,21 +17,13 @@ const Editor = forwardRef<MakeCodeFrameDriver, EditorProps>(function Editor(
   props,
   ref
 ) {
-  const logging = useLogging();
-  const { project, editorCallbacks } = useProject();
-  const initialProjects = useCallback(() => {
-    logging.log(
-      `[MakeCode] Initialising with header ID: ${project.header?.id}`
-    );
-    return Promise.resolve([project]);
-  }, [logging, project]);
```

Comment:
I just moved this into useProjects as it seemed odd that only one aspect of the MakeCode integration wasn't in that hook.